index.html

```diff
@@ -147,12 +155,39 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           purpose is to be handed to {{BrowserCaptureMediaStreamTrack/cropTo}} as input.
         </p>
         <pre class="idl">
-          [Exposed = Window]
+          [Exposed = Window, Serializable]
           interface CropTarget {
             // Intentionally empty; just an opaque identifier.
           };
         </pre>
         <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+
+        <p id="serializable-croptarget">
+          {{CropTarget}} objects are [= serializable objects =] [[!HTML]]. Their [= serialization
+          steps =], given <var>value</var> and <var>serialized</var>, are:
+        </p>
+        <ol>
+          <li>
+            Set <var>serialized</var>.[[\Data]] a randomly generated, globally unique {{DOMString}}.
+          </li>
+        </ol>
+        <p>
+          Their <a>deserialization steps</a>, given <var>serialized</var> and <var>value</var>, are:
+        </p>
+        <ol>
+          <li>
+            Compare <var>serialized</var>.[[\Data]] against a record of all {{CropTargets}} produced
+            and serialized thus far in the current top-level [=browsing context=], or in any of its
+            descendent [=browsing contexts=].
+            <ol>
+              <li>
+                If <var>serialized</var>.[[\Data]] was obtained by serializing previous
+                {{CropTarget}}, then set <var>value</var> to an equivalent {{CropTarget}}.
```

Comment:
This equivalent has a fuzzy definition.
That is one reason why I think we should introduce an element reference slot in the spec.

Reply:
I think [this comment](https://github.com/w3c/mediacapture-region/pull/32#issuecomment-1080798232) applies here too; only its second half is relevant, so I'll repeat:
It might be that using weak references is standard procedure and would be well understood by readers. If that is the case, then I do not wish to delay us further. Could you point out an example of a Transferable/Serializable object that stores a weak reference to an Element in the original context, and where the weak reference gets replicated in the new context?